metals/src/main/scala/scala/meta/internal/metals/Tables.scala

```diff
@@ -96,12 +97,17 @@ final class Tables(
       if (isAutoServer) ";AUTO_SERVER=TRUE"
       else ""
     val dbfile = workspace.resolve(".metals").resolve("metals")
+    val oldDbfile = workspace.resolve(".metals").resolve("metals.h2.db")
```

Comment:
Won't this be found on ever re-launch of metals? So then it will delete and recreate every restart?

Reply:
After h2 update db file will have a new name - `metals.mv.db`

Reply:
Actually, due to a change of database the name also changes, wo I am only deleting the old database.

Reply:
ahhhh ok, that makes sense then.

Reply:
maybe the new database name is worth mentioning in the comment?

Reply:
added the comment!